src/main/java/com/heeverse/concert/controller/ConcertController.java

```diff
@@ -0,0 +1,54 @@
+package com.heeverse.concert.controller;
+
+import com.heeverse.concert.dto.ConcertRequestDto;
+import com.heeverse.concert.exception.ConcertTimeValidationException;
+import com.heeverse.concert.service.ConcertService;
+import jakarta.validation.Valid;
+import java.time.LocalDateTime;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.util.ObjectUtils;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ * @author jeongheekim
+ * @date 2023/08/07
+ */
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "/concert")
```

Comment:
어노테이션에서 value는 생략이 가능합니다.
```suggestion
@RequestMapping("/concert")
```